cmd/pbapi/main.go

```diff
@@ -64,11 +64,11 @@ func main() {
 	// Routes for metrics
 	mr := chi.NewRouter()
 	mr.Get("/", statusOk)
-	mr.Handle("/metrics", promhttp.Handler())
+	mr.Handle(config.Prometheus.Path, promhttp.Handler())
 
-	log.Info().Int("webPort", 8000).Int("metricsPort", config.Prometheus.Port).Msg("Starting new instance")
+	log.Info().Int("webPort", config.Application.Port).Int("metricsPort", config.Prometheus.Port).Msg("Starting new instance")
```

Comment:
Could you actually drop these ints from the logging message here? Or explain what is the benefit. Structured logs are good for searching (give me all logs for a particular request, account, user, organization etc).